index.d.ts

```diff
@@ -1725,6 +1725,16 @@ declare namespace Eris {
     category_id: number;
   }
 
+  interface WelcomeChannel {
+	channelID: string;
+	description: string;
+	emojiID: string;
+	emojiName: string;
+  }
+  interface WelcomeScreen {
+	description: string;
+	welcomeChannels: WelcomeChannel[];
+  }
```

Comment:
Needs to go in its relevant interface group a little while up

Reply:
That seems to belong here though. This is a Guild interface isn't it?